packages/tokens/src/figma-tokens/_docs.json

```
@@ -851,13 +915,125 @@
+          "value": "Read-only",
+          "type": "text"
+        },
+        "description": {
+          "value": "Add the read-only attribute to draw a read-only select.",
+          "type": "text"
+        }
+      },
+      "disabled": {
+        "title": {
+          "value": "Disabled",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the disabled attribute to disable a select.",
+          "type": "text"
+        }
+      },
+      "multiple": {
+        "title": {
+          "value": "Multiple",
+          "type": "text"
+        },
+        "description": {
+          "value": "To allow multiple options to be selected, use the multiple attribute. It’s a good practice to use clearable when this option is enabled. To set multiple values at once, set value to a space-delimited list of values.",
+          "type": "text"
+        }
+      },
+      "initialvalue": {
+        "title": {
+          "value": "Setting Initial Values",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the value attribute to set the initial selection.\n\nWhen using multiple, the value attribute uses space-delimited values to select more than one option. Because of this, <sl-option> values cannot contain spaces. If you’re accessing the value property through Javascript, it will be an array.",
+          "type": "text"
+        }
+      },
+      "group": {
+        "title": {
+          "value": "Grouping Options",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use <syn-optgroup> to group listbox items visually.",
+          "type": "text"
+        }
+      },
+      "size": {
+        "title": {
+          "value": "Sizes",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the size attribute to change a select’s size. Note that size does not apply to listbox options.",
+          "type": "text"
+        }
+      },
+      "prefix": {
+        "title": {
+          "value": "Prefix Suffix Text and Icons",
+          "type": "text"
+        },
+        "description": {
+          "value": "Use the prefix and suffix slots to add text and icons.",
+          "type": "text"
+        }
+      },
+      "gettag": {
+        "title": {
+          "value": "Custom Tags",
+          "type": "text"
+        },
+        "description": {
+          "value": "When multiple options can be selected, you can provide custom tags by passing a function to the getTag property. Your function can return a string of HTML, a Lit Template, or an HTMLElement. The getTag() function will be called for each option. The first argument is an <sl-option> element and the second argument is the tag’s index (its position in the tag list).\n\nRemember that custom tags are rendered in a shadow root. To style them, you can use the style attribute in your template or you can add your own parts and target them with the ::part() selector.",
```

`<sl-option>` should be exchanged with `<syn-option>`

```suggestion
          "value": "When multiple options can be selected, you can provide custom tags by passing a function to the getTag property. Your function can return a string of HTML, a Lit Template, or an HTMLElement. The getTag() function will be called for each option. The first argument is an <syn-option> element and the second argument is the tag’s index (its position in the tag list).\n\nRemember that custom tags are rendered in a shadow root. To style them, you can use the style attribute in your template or you can add your own parts and target them with the ::part() selector.",
```